### ISSUE TITLE
better variable names

### ISSUE THREAD

**Issue by armfazh** [mmcloughlin/addchain#118]
it is noisy to get variables named in binary `_100` or `_101`.
Better to follow a standard rule, such as `i4` or any other counter in decimal or hex.
or at least, allow the user to specify the format desired.

**Comment by mmcloughlin** [comment]
Yeah I chose that to match https://briansmith.org/ecc-inversion-addition-chains-01. But yes it's not the best for code generation. I think your request could be covered under #94.

How did you generate code from the output? By hand?
